shared/chat/conversation/container.js

```diff
@@ -1,48 +1,40 @@
 // @flow
 import * as Constants from '../../constants/chat'
-import * as Creators from '../../actions/chat/creators'
 import Conversation from './index'
-import {List} from 'immutable'
 import {connect} from 'react-redux'
 import {navigateAppend} from '../../actions/route-tree'
 import {withState, withHandlers, compose, branch, renderNothing, lifecycle} from 'recompose'
 
 import type {Props} from '.'
 import type {TypedState} from '../../constants/reducer'
 
-type StateProps = {
+type StateProps = {|
   finalizeInfo: ?Constants.FinalizeInfo,
-  listScrollDownCounter: number,
   rekeyInfo: ?Constants.RekeyInfo,
-  participants: List<string>,
   selectedConversationIDKey: ?Constants.ConversationIDKey,
   showLoader: boolean,
   supersededBy: ?Constants.SupersedeInfo,
   supersedes: ?Constants.SupersedeInfo,
-}
+|}
 
-type DispatchProps = {
+type DispatchProps = {|
```

Comment:
I'm starting to type what the mapDispatch and mapProps return for better flow typing